packages/jest-preset-mc-app/jest-preset.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
     resolveRelativePath('./setup-tests.js'),
     'jest-localstorage-mock',
   ],
-  setupFilesAfterEnv: [resolveRelativePath('./setup-test-framework.js')],
+  setupFilesAfterEnv: [resolveRelativePath('./setup-test-framework.ts')],
```

Comment:
Hmm this should stay `.js` otherwise it breaks for whoever is using the package without typescript 🤔